### PR TITLE
OCM-9690 | test: update id:74387,64078

### PR DESCRIPTION
**Pull request by aaraj7**
[OCM-8933](https://issues.redhat.com//browse/OCM-8933)
$ ginkgo --focus 74387 tests/e2e/
Ginkgo detected a version mismatch between the Ginkgo CLI and the version of Ginkgo imported by your packages:
  Ginkgo CLI Version:
    2.14.0
  Mismatched package versions found:
    2.17.1 used by e2e

  Ginkgo will continue to attempt to run but you may see errors (including flag
  parsing errors) and should either update your go.mod or your version of the
  Ginkgo CLI to match.

  To install the matching version of the CLI run
    go install github.com/onsi/ginkgo/v2/ginkgo
  from a path that contains a go.mod file.  Alternatively you can use
    go run github.com/onsi/ginkgo/v2/ginkgo
  from a path that contains a go.mod file to invoke the matching version of the
  Ginkgo CLI.

  If you are attempting to test multiple packages that each have a different
  version of the Ginkgo library with a single Ginkgo CLI that is currently
  unsupported.
  
Running Suite: ROSA CLI e2e tests suite - /home/aaraj/ocm-forked-code/rosa/tests/e2e
====================================================================================
Random Seed: 1721311915

Will run 1 of 139 specs
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS•SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS

Ran 1 of 139 Specs in 501.586 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 138 Skipped
PASS

Ginkgo ran 1 suite in 8m36.366307228s
Test Suite Passed
[aaraj@aaraj-thinkpadt14

### CODE REVIEW

File: tests/e2e/test_rosacli_node_pool.go [hunk 1]
```diff
@@ -1214,6 +1256,44 @@ var _ = Describe("Edit nodepool",
...
+						"Updated machine pool '%s' on hosted cluster '%s'",
+						machinePoolName,
+						clusterID))
+
+				By("Describe the nodepool to see max surge and max unavailable is set correctly")
+				out, err = machinePoolService.DescribeAndReflectNodePool(clusterID, machinePoolName)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(out.ManagementUpgrade[1]["Max surge"]).
+					To(
+						Equal("7"))
```
Comment (yasun1) [inline tests/e2e/test_rosacli_node_pool.go:1277]: The style is weird.

Comment (aaraj7) [reply tests/e2e/test_rosacli_node_pool.go:1277]: But can we leave it the same I guess ?

Comment (aaraj7) [reply tests/e2e/test_rosacli_node_pool.go]: @yasun1  I changed it, please approve

**Comment by aaraj7** [comment]
@yasun1 I updated the test case 64078 too, PTAL, Thanks!

**Pull request merged**